app/views/categories/_expense_item.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="item-card">
+    <div class="item-meta">
+        <div class="item-icon">💸</div>
+        <div class="item-group">
+            <h3 class="item-name"><%= expense.name %></h3>
+            <p class="item-data">
+                <%= expense.updated_at.to_datetime.strftime("%B %d, %Y") %>
+            </p>
+        </div>
+    </div>
+    <div class="item-group">
+        <p class="item-value">
```

Comment:
- [ ] On the home page, the total amount spent on a given category is not shown. It only shows $0.00. Please make sure that the total amount spent on a particular category is shown on the home page :+1: 
![image](https://github.com/uchexm/Spendrit/assets/45460192/2e6a2696-b550-40b2-b8d1-18eb479a1e11)

Reply:
On it ✌

Reply:
On it ✌